Header header
int32 uavNum
int32 rqstrId
int32 rspdrId
geometry_msgs/Vector3 rqstrVel 
geometry_msgs/Vector3 rspdrVel
geometry_msgs/Vector3 rlpos_RqstToRspdr
float64 distance